styles/scss/image-gallery.scss

```diff
@@ -18,6 +18,9 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
   @include vendor-prefix('user-select', none);
   -webkit-tap-highlight-color: $ig-transparent;
 
+  &.image-gallery-rtl {
+    direction: rtl;
```

Comment:
When setting the `isRTL` is set to true and setting `thumbnailPosition` to `left|right` there is a missing margin that was previously on the opposite side. See SS below.

## `isRTL={false} thumbnailPosition='right'`
![screen shot 2018-03-15 at 11 42 49 pm](https://user-images.githubusercontent.com/1286776/37502742-eff9d4ac-28aa-11e8-9600-39ea855876d1.png)

## `isRTL={true} thumbnailPosition='left'`
![screen shot 2018-03-15 at 11 43 00 pm](https://user-images.githubusercontent.com/1286776/37502754-04dba7f6-28ab-11e8-8c6b-3e9eb1b5a176.png)